types/module/module.go

```diff
@@ -217,6 +218,10 @@ func (m *Manager) SetOrderInitGenesis(moduleNames ...string) {
 			return !hasABCIGenesis
 		}
 
+		if _, hasGenesis := module.(appmodulev2.HasGenesis); hasGenesis {
+			return !hasGenesis
+		}
+
 		_, hasGenesis := module.(HasGenesis)
```

Comment:
Shouldn't we drop the support of this one? And removing it from types/modules.

Then the new genesis api gets aliased in appmodule (v1).

Reply:
removed them